frontend/src/Admin/components/EditHometown.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import {
+  Form,
+  FormGroup,
+  FormFeedback,
+  Label,
+  Button,
+  Row,
+  Col
+} from 'reactstrap'
+import { Formik, Field } from 'formik'
+import yup from 'yup'
+
+class EditHometown extends Component {
+  static propTypes = {
+    create: PropTypes.func.isRequired,
+    handleError: PropTypes.func.isRequired,
+    student: PropTypes.shape({
+      username: PropTypes.string.isRequired,
+      firstName: PropTypes.string.isRequired,
+      lastName: PropTypes.string.isRequired,
+      hometown: PropTypes.string
+    }).isRequired
+  }
+
+  renderErrors = (touched, errors, field) => {
+    // Render feedback if this field's been touched and has errors
+    if (touched[field] && errors[field]) {
+      return (
+        <FormFeedback style={{ display: 'block' }}>
+          {errors[field]}
+        </FormFeedback>
+      )
+    }
+  }
+
+  render () {
+    const { create, handleError, student } = this.props
+
+    return (
+      <Formik
+        initialValues={{
+          username: student.username,
+          firstName: student.firstName,
+          lastName: student.lastName,
+          hometown: student.hometown,
+          isEditing: 'no',
+        }}
+        validationSchema={yup.object().shape({
+          username: yup.string().required('Required'),
+          firstName: yup.string().required('Required'),
+          lastName: yup.string().required('Required'),
+          hometown: yup.string().nullable()
```

Comment:
Do we want to let the Admin just delete a hometown? Should they have to give it a value?

Reply:
I don’t see why not, they are an admin

Reply:
It doesn’t break anything if they do. As far as I can tell it just asks the user for their hometown again, which might be the best way to do it

Reply:
Fair.